opentracing-spring-cloud-starter-zipkin/README.md

```diff
@@ -0,0 +1,92 @@
+# Zipkin OpenTracing Spring Boot starter
+
+This starter provides a single entry point for Spring Boot application to use in order to setup OpenTracing instrumentation 
+
+## Configuration
+
+```xml
+<dependency>
+  <groupId>io.opentracing.contrib</groupId>
+  <artifactId>opentracing-spring-cloud-starter-zipkin</artifactId>
+</dependency>
+```
+
+Simply adding the dependency to the application will ensure that all required OpenTracing and Zipkin dependencies are transitively resolved.
+Furthermore the dependency will ensure that Spring Boot will auto configure all the necessary OpenTracing beans when the application starts.
+
+
+The only required property that needs to be set is the Zipkin collector URL:
+
+```
+opentracing.zipkin.http-sender.url=http://somehost:someport/somepath
```

Comment:
See my comment about default url, This can stay here, however change it please to http://localhost:9411/api/v2/spans

Reply:
Also mention that this repository uses JSON_V2